### PR TITLE
[VL] Add parquet write benchmark

### PR DESCRIPTION
**Pull request by JkSelf**
## What changes were proposed in this pull request?
This PR add the parquet writer benchmark in velox backend.

(Fixes: \#ISSUE-ID)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

File: cpp/velox/benchmarks/ParquetWriteBenchmark.cc [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+ * limitations under the License.
+ */
+
+#include <arrow/c/abi.h>
+#include <arrow/c/bridge.h>
+#include <arrow/filesystem/filesystem.h>
+#include <arrow/io/interfaces.h>
+#include <arrow/memory_pool.h>
+#include <arrow/record_batch.h>
+// #include <arrow/testing/gtest_util.h>
```
Comment (jinchengchenghh) [inline cpp/velox/benchmarks/ParquetWriteBenchmark.cc:24]: remove this comment line

File: cpp/velox/benchmarks/ParquetWriteBenchmark.cc [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+      ->Iterations(iterations)
+      ->Threads(threads)
+      ->ReportAggregatesOnly(false)
+      ->MeasureProcessCPUTime()
+      ->Unit(benchmark::kSecond);
+
+  benchmark::Initialize(&argc, argv);
+  benchmark::RunSpecifiedBenchmarks();
+  benchmark::Shutdown();
+}
```
Comment (jinchengchenghh) [inline cpp/velox/benchmarks/ParquetWriteBenchmark.cc:352]: Please add empty line at the end of file

Comment (JkSelf) [reply cpp/velox/benchmarks/ParquetWriteBenchmark.cc]: Done.

File: cpp/velox/benchmarks/ParquetWriteBenchmark.cc (hunk 1 above)
Comment (JkSelf) [reply cpp/velox/benchmarks/ParquetWriteBenchmark.cc]: Done.

**Comment by JkSelf** [comment]
@jinchengchenghh Do you have any further comment?

**Review by jinchengchenghh — APPROVED** [review]

**Pull request merged**